.circleci/config.yml

```diff
@@ -125,16 +129,13 @@ commands:
                 command: pip install tox && tox -e 'wait' << parameters.wait >>
       - run:
           name: "Running tests matching '<< parameters.pattern >>'"
-          command: "riot -v run -s '<< parameters.pattern >>'"
+          command: "echo -p2.7,-p3.5,-p3.6,-p3.7,-p3.8,-p3.9 | tr ',' '\n' | circleci tests split | xargs riot -v run -s '<< parameters.pattern >>'"
```

Comment:
same comment as the other PR, I wonder if we should just build `CIRCLECI_NODE_INDEX` and `CIRCLECI_NODE_TOTAL` into `riot`, could make this much easier to manage.

Reply:
Agree this is best done within riot!